src/tc_classify_kern.c

```diff
@@ -177,8 +181,35 @@ __u32 get_ipv4_addr(struct __sk_buff *skb, __u32 l3_offset, __u32 ifindex_type)
 	default:
 		ipv4 = 0;
 	}
+	key->address.in6_u.u6_addr32[3] = ipv4;
+}
+
+
+static __always_inline
+void get_ipv6_addr(struct __sk_buff *skb, __u32 l3_offset, __u32 ifindex_type,
+		struct ip_hash_key *key)
+{
+	void *data_end = (void *)(long)skb->data_end;
+	void *data     = (void *)(long)skb->data;
+	struct ipv6hdr *ip6h = data + l3_offset;
+
+	if (ip6h + 1 > data_end) {
+		//bpf_debug("Invalid IPv6 packet: L3off:%llu\n", l3_offset);
+		//*dst = nulladdr;
```

Comment:
What is this leftover code in the comment "nulladdr".
Please remove that comment

Reply:
Resolved by https://github.com/xdp-project/xdp-cpumap-tc/pull/14/commits/ac81223309022e7375b6ae341c92ca79ebefc314